CONTRIBUTING.md

```diff
@@ -1,4 +1,37 @@
 # Contributing guidelines
 
-This project is in its very early stages and is not actively soliciting
-contributions at the present time.
+If you're looking for something to work on, please look at our 
+[milestones](https://github.com/kubernetes/cluster-registry/milestones) or our
+[open issues](https://github.com/kubernetes/cluster-registry/issues)
+and assign an issue to yourself.
+
+## How to become a contributor and submit your own code
+
+### Contributor License Agreements
+
+We'd love to accept your patches! Before we can take them, we have to jump a
+couple of legal hurdles.
+
+Please fill out either the individual or corporate Contributor License Agreement
+(CLA).
+
+  * If you are an individual writing original source code and you're sure you
+    own the intellectual property, then you'll need to sign an
+    [individual CLA](https://identity.linuxfoundation.org/node/285/node/285/individual-signup).
+  * If you work for a company that wants to allow you to contribute your work,
+    then you'll need to sign a
+    [corporate CLA](https://identity.linuxfoundation.org/node/285/organization-signup).
+
+Follow either of the two links above to access the appropriate CLA and
+instructions for how to sign and return it. Once we receive it, we'll be able
+to accept your pull requests.
+
+### Getting started with development
+
+Please refer to the [developer documentation](/docs/development.md) to get
```

Comment:
nit: the link says 'developer' but this is called 'development' everywhere else.

Reply:
Good eye, fixed.